### ISSUE TITLE
Can't use grunt responsive images

### ISSUE THREAD

**Issue by henrybarnacle** [andismith/grunt-responsive-images#122]
### Expected Behaviour
entering 'npm install grunt-responsive-images --save-dev' into the terminal should add responsive images grunt task..

### Actual Behaviour
It won't work! Instead i get this message..
npm WARN reponsive-images@0.1.6 No license field

**Comment by andismith** [comment]
Are you sure it's not installing? This looks like it's just a warning.

Looks like the license value has changed recently so I've updated that.

Can you see if the package is installing in to node_modules and if not paste the full error?

Thanks,
Andy

**Comment by henrybarnacle** [comment]
when I type in the terminal..
node -v
I get..
v6.9.1
Then..
grunt -version
I get..
grunt-cli v1.2.0
grunt v1.0.1
so these seem to be in place.
Then in my project directory..
npm install grunt-responsive-images --save-dev
I get..
reponsive-images@0.1.0 /Users/henrybarnacle/Desktop/Project-Part-1-Start
└── grunt-responsive-images@0.1.9 

npm WARN reponsive-images@0.1.0 No license field.

At this point it has given me the node_modules folder in my project but both the WARN message and the grunt-responsive-images@0.1.9 message are in yellow and doesn't seem to have done all it's supposed to do.
I've tried adding a license field in the JSON and repeating these steps and I still get the grunt-responsive-images@0.1.9 message in yellow but nothing else.
At this point if I type grunt in to the terminal I get..
Running "responsive_images:dev" (responsive_images) task
Warning: Could not execute GraphicsMagick/ImageMagick: identify "-ping" "-format" "%m:%T:%s" "images_src/cockatoos.jpg" this most likely means the gm/convert binaries can't be found Use --force to continue.

Aborted due to warnings.

Any help would be greatly appreciated, I've tried everything I could think of!

**Comment by andismith** [comment]
Is your project called 'reponsive-images' by any chance? Otherwise, I'm not sure where this is coming from as this project is called 'grunt-responsive-images' (no typo). If it is, then it's complaining about your project not having a license field I think.

As for the could not execute GM error, please make sure you have Graphicsmagick installed as per the getting started guide.

Hope this helps

**Comment by henrybarnacle** [comment]
Oh sorry for the typo, no it's called 'responsive-images'
Graphicsmagick is definitely installed, just tried to re-install..

Warning: graphicsmagick-1.3.25 already installed

I've tried adding different license fields, it still doesn't work.
Thanks for trying to help-think I've got to accept defeat at this point and maybe come back to it another time, Henry

**Comment by henrybarnacle** [comment]
Finally got grunt working, first by adding a license field, then I was still getting this message-

Warning: Could not execute GraphicsMagick/ImageMagick: identify "-ping" "-format" "%m:%T:%s" "images_src/cockatoos.jpg" this most likely means the gm/convert binaries can't be found Use --force to continue.

I had already installed Graphicsmagick several times..I hadn't realised you needed to download both Graphicsmagick AND Imagemagick! 
Now I've installed Imagemagick also and grunt now works perfectly, thank you!
